packages/rocket-webhook-types/src/webhook-params.ts

```diff
@@ -9,13 +9,19 @@ export type WebhookAuthorizer = (currentUser?: UserEnvelope, request?: WebhookRe
 
 export type WebhookRoleAccess = 'all' | Array<Class<RoleInterface>> | WebhookAuthorizer
 
+export enum AllowedHttpMethods {
+  GET = 'GET',
+  POST = 'POST',
+}
```

Comment:
Maybe I'm a bit nitpicky here, but I would use a singular name for the enum `AllowedHttpMethod` , so that in `webhook-dispatcher.ts` you can use this code instead:
```typescript
    if (!allowedMethods.includes(request.req.method as AllowedHttpMethod)) {
      return WebhookResponse.responseFailure(new Error(`Method ${request.req.method} not allowed`))
    }
```

Reply:
Good suggestion. Changed in commit edffcb5.